### PR TITLE
chore(#1169)Add page on how to run multiple chrome versions

### PR DESCRIPTION
**Pull request by lorerod**

### CODE REVIEW

File: content/en/contribute/code/core/run-multiple-chrome-versions.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  How to run multiple Chrome versions on your local machine
+aliases: >
+  /core/guides/multiple-chrome-versions
+---
+
+{{% alert title="Note" %}}
+These steps are suitable for Mac. It was tested in a Mac Intel.  It can be adapted to any Chrome version.
+{{% /alert %}}
+
+Follow these steps on a Mac to run Chrome version 90 besides your actual up to date Chrome app.
```
Comment (latin-panda) [inline content/en/contribute/code/core/run-multiple-chrome-versions.md:15]:
```suggestion
Follow these steps on a Mac to run Chrome version 90 while having another Chrome app on a different version. 
```
I'm not a native speaker, but `besides your actual` sounds weird to me 🤔

File: content/en/contribute/code/core/run-multiple-chrome-versions.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- Download Chrome `90.0.4430.72` from [slimjet](https://www.slimjet.com/chrome/google-chrome-old-version.php?cmtx_sort=)
+- Do not install the `Google Chrome.app` in your `Application` folder. Install it in your `Desktop` folder for example.
+- Change the name of the app to `Google Chrome 90.app` and then move it to the `Application` folder, ***without*** overwritting your current Chrome
+- Remove Chrome’s automatic updates by:
+  - Right click on `Application/Google Chrome 90.app`
```
Comment (latin-panda) [inline content/en/contribute/code/core/run-multiple-chrome-versions.md:20]:
```suggestion
- Remove Chrome’s automatic updates by:
  - Close all Chrome open instances.
  - Open `Google Chrome 90.app` while not having Internet, so it doesn't give an error after modifying the `Info.plis`.
  - Right-click on `Application/Google Chrome 90.app`
```
This step worked better for me.

File: content/en/contribute/code/core/run-multiple-chrome-versions.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  - Click on `Show package content`
+  - Open in the IDE the file `Contents/Info.plist`
```
Comment (latin-panda) [inline content/en/contribute/code/core/run-multiple-chrome-versions.md:22]:
```suggestion
  - Open the file `Contents/Info.plist` in your IDE 
```

**Review by latin-panda — CHANGES_REQUESTED** [review]
This is super helpful! Thanks Lorena

File: content/en/contribute/code/core/run-multiple-chrome-versions.md (hunk 2 above)
Comment (lorerod) [reply content/en/contribute/code/core/run-multiple-chrome-versions.md:20]: Add extra steps to remove Chrome´s automatic update.

**Review by latin-panda — APPROVED** [review]
Nice!!

**Pull request merged**